egs/librispeech/s5/run.sh

```diff
@@ -355,6 +355,10 @@ local/run_cleanup_segmentation.sh
 # train NN models on the entire dataset
 local/nnet2/run_7a_960.sh
```

Comment:
@freewym Please update the RESULTS file. Remember to add the command to be run to generate the results, beside each set of results.